tests/basic/commands

```diff
@@ -35,6 +35,10 @@ test tests/basic/undefinedReferences.ref get undefinedReferences
 test tests/basic/uniqueIpAssignments.ref get uniqueIpAssignments
 test tests/basic/unusedStructures.ref get unusedStructures
 
+# example2 testrig
+test tests/basic/init-example2.ref init-testrig test_rigs/example2
+test tests/basic/inferroles.ref get inferRoles
```

Comment:
I'm not sure if this is what you're referring to, but currently there is a naming inconsistency -- we use camel (e.g., inferRoles) at the command line (and for the names of template questions), but the getName() method of a Question must be all lower case (e.g., inferroles) or else the Java client breaks (since it converts question-name arguments to lowercase before looking them up).  That bug took me a while to find.